src/media/js/previews.js

```diff
@@ -178,8 +178,15 @@ define('previews',
 
         $tray.css('width', z.win.width() + 'px');
         var offset = $tray.offset();
-        desktopMarginLeft += -1 * offset.left;
-        $tray.css('margin-left', desktopMarginLeft + 'px');
+
+        // LTR or RTL?
+        if ($('body.html-rtl').length) {
```

Comment:
Though there would be a small perf gain in not setting the class on the first place as well